aspnetcore/blazor/hosting-models.md

```diff
@@ -103,6 +103,189 @@ The graph is rerendered, and a UI *diff* (difference) is calculated. This diff i
 
 A component is disposed after the user navigates away from it on the client. While a user is interacting with a component, the component's state (services, resources) must be held in the server's memory. Because the state of many components might be maintained by the server concurrently, memory exhaustion is a concern that must be addressed. For guidance on how to author a Blazor Server app to ensure the best use of server memory, see <xref:security/blazor/server>.
 
+### Integrate Razor components into Razor Pages and MVC apps
+
+#### Use components in pages and views
+
+An existing Razor Pages or MVC app can integrate Razor components into pages and views:
+
+1. In the app's layout file (*_Layout.cshtml*):
+
+   * Add the following `<base>` tag to the `<head>` element:
+
+     ```html
+     <base href="~/" />
+     ```
+
+     The `href` value (the *app base path*) in the preceding example assumes that the app resides at the root URL path (`/`). For more information on setting the app base path, see <xref:host-and-deploy/blazor/index#app-base-path>.
+
+     The *_Layout.cshtml* file is located in the *Pages/Shared* folder in a Razor Pages app or *Views/Shared* folder in an MVC app.
+
+     If the app is a sub-application, follow the guidance in the *App base path* section of the <xref:host-and-deploy/blazor/index#app-base-path> article.
```

Comment:
I suggest moving this above a bit where you're already talking about the base path. See my suggestion above.